pkg/term/term_windows.go

```diff
@@ -33,7 +33,7 @@ const (
 // vtInputSupported is true if enableVirtualTerminalInput is supported by the console
 var vtInputSupported bool
 
-// StdStreams returns the standard streams (stdin, stdout, stedrr).
+// StdStreams returns the standard streams (stdin, stdout, stderr).
```

Comment:
I also found that in file `pkg/term/term.go#L34` there is a same typo.
Could you please combine that in this single PR?